src/ui/gtk/settings_cb.c

```diff
@@ -951,7 +959,7 @@ void
 on_entry_dbg_property_pattern_activate(GtkEditable *unused_editable,
 	gpointer unused_udata)
 {
-	static gchar old_pattern[1024];
+	static gchar old_pattern[1024] = {0};
```

Comment:
Can you put spaces around braces and say:
`{ '\0' }`
to make it more obvious that we are initializing an array of characters